pages/services/[ware].js

```diff
@@ -1,13 +1,19 @@
 import { useRouter } from 'next/router'
 import { Error, ItemPage, Loading } from '@scientist-softserv/webstore-component-library'
-import { configureErrors, DEFAULT_WARE_IMAGE, useOneWare } from '../../utils'
+import {
+  configureErrors, DEFAULT_WARE_IMAGE, SHOW_SERVICE_PAGE, useOneWare 
+} from '../../utils'
 
 const Service = () => {
   const router = useRouter()
   const { id } = router.query
   const { ware, isLoading, isError } = useOneWare(id)
 
   if (isError) return <Error errors={configureErrors([isError])} router={router} />
+  
+  if (SHOW_SERVICE_PAGE === false) {
```

Comment:
would it be better to just not have a link rather thank render an error?

Reply:
if SHOW_SERVICE_PAGE is false, the link leads to a different path (it directs the user to create a new request instead), and there will be nowhere on the site that has a link to the path for this page.

the error is just in case of the very slim chance someone ends up on this path. The error that renders is identical to what the user would see if they hit a 404 path.